### PR TITLE
feat(web): add basic layout for the web app

### PR DESCRIPTION
**Pull request by xiaohai-huang**
Use flexbox to structure the web app.

### CODE REVIEW

File: web/pages/components/Layout/Layout.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  { label: "Lines", link: "/lines" },
+  { label: "Game Grid", link: "/game-grid" },
+  { label: "Stats", link: "/stats" },
+  { label: "Charts", link: "/charts" },
+  { label: "Blog", link: "/blog" },
+];
+
+function Layout({ children }: LayoutProps) {
+  const router = useRouter();
+  console.log(router.pathname);
```
Comment (FHFHockey-dev) [inline web/pages/components/Layout/Layout.tsx:25]: Can you remove this debugging log?

**Review by FHFHockey-dev — CHANGES_REQUESTED** [review]

File: web/pages/components/Layout/Layout.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.navBarLeftSide}>
+        <nav>
+          <ul>
+            {/* Links in Nav Bar ("currentPage" on whichever page you're on) -----------*/}
```
Comment (FHFHockey-dev) [inline web/pages/components/Layout/Layout.tsx:32]: These links are just placeholder. Write some comments to denote that.

**Review by FHFHockey-dev — CHANGES_REQUESTED** [review]

File: web/pages/components/Layout/Layout.tsx (hunk 1 above)
Comment (xiaohai-huang) [reply web/pages/components/Layout/Layout.tsx]: sure

File: web/pages/components/Layout/Layout.tsx (hunk 2 above)
Comment (xiaohai-huang) [reply web/pages/components/Layout/Layout.tsx]: ok

**Review by FHFHockey-dev — APPROVED** [review]
Very good, mate

**Pull request merged**